aws/iam_test.go

```diff
@@ -193,3 +194,87 @@ func TestIAMParseError(t *testing.T) {
 	_, err := provider.GetCertificates()
 	require.Equal(t, ErrNoCertificates, err)
 }
+
+func TestIAMTagFiltering(t *testing.T) {
+	foobarNotBefore := time.Date(2017, 3, 29, 16, 11, 32, 0, time.UTC)
+	foobarNotAfter := time.Date(2027, 3, 27, 16, 11, 32, 0, time.UTC)
```

Comment:
2027 -> 3027 ?

Reply:
Yeah, its getting close hehe

Reply:
It seems this is not a problem, we only evaluate if the cert is still valid in the matching phase. These is another test in the same file with `NotAfter ~= 2018`, so I guess its fine.

Reply:
ah we also have a fixed cert so all data is fixed, true.